README.md

```diff
@@ -19,9 +20,9 @@ Docker images with this release tag will be built automatically by GitHub Action
 
 * push to ghcr like:
 
-`docker tag autonity:latest ghcr.io/clearmatics/services-testnet-agc:v0.8.0-bakerloo01`
+`docker tag services-testnet-agc:latest ghcr.io/clearmatics/services-testnet-agc:v0.8.4-bakerloo`
```

Comment:
I would suggest to still number after `bakerloo`, ie `bakerloo01`, since we know from experience that we might need to create a new one from the same version of Autonity, e.g. to change some config switch, so it makes sense to number hose.